### ISSUE TITLE
Beginner Questions

### ISSUE THREAD

**Issue by sangyoo91** [arunoda/meteor-up-legacy#548]
Is it possible I use this for locally run apps? I am making an app for a friend for his own use in his small company.

If so, would there be an advantage using this over `meteor run` on console?

**Comment by MasterJames** [comment]
It's meant for cloud deployment.
the real purpose in my mind is to run 5 machines for total fault tolerance with rolling updates etc.
internally you would not need what mup and mupx provide unless you need it to never fail and always be running for many people simultaneously.

**Comment by sangyoo91** [comment]
@MasterJames Yes! That's exactly what I was meaning to use it for (2 machines). Are there any references to set this up locally for multiple machines?
